.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ * ]
```

Comment:
@leecalcote it should be `**`, as `*` does not include branches with  `/` in their name.
if this is changed, we're good to go :+1: 

`master` in `pull_request` is fine, it is a subset and afaik we don't create PRs to other branches.
(can't seem to edit this file or push to this branch, otherwise I would have)

Reply:
@mgfeller I think this was the last excuse we needed to graduate you from Meshery Adapter Maintainer to Meshery Maintainer. 💪

Reply:
Second `*` added. Let's see how she works...

Reply:
> @mgfeller I think this was the last excuse we needed to graduate you from Meshery Adapter Maintainer to Meshery Maintainer. muscle

Cool :smile: Graduation party?! :tada: